src/evolutionaryAlgorithm/Observers/impl/MinFitnessFunctionObserver.py

```diff
@@ -5,22 +5,23 @@
 
 
 class MinFitnessFunctionObserver(ObserverInterface):
-    fileName = "minFitnessFunctionObserver.txt"  # TODO change this
-    open(fileName, 'w').close()
 
-    @classmethod
-    def observe(cls, population: List[IndividualInterface], step: int):
+    def __init__(self, path: str) -> None:
+        self.fileName = path
+        with open(self.fileName, 'w') as f:
+            f.write("")
+
+    def observe(self, population: List[IndividualInterface], step: int):
         evaluationMin = 999999  # TODO change this
```

Comment:
tutaj wystarczy zrobić evluationMin = population[0].getEvaluation() i guess

Reply:
a przynajmniej ja tak zrobiłem jak szukałem minimum w którymś miejscu i no po prostu to chyba lepiej będzie wyglądało

Reply:
Done